### PR TITLE
Deleting quotes - Issue 228

### PR DESCRIPTION
**Pull request by ankritisingh**
It doesn't run with the quotes around r(table)

### CODE REVIEW

**Comment by kbjarkefur** [comment]
This is related to issue #228. It is supposed to be a transpose, so the last quote is indeed correct. Do not merge this PR before that is corrected

File: code/code.do [hunk 1]
```diff
@@ -5,7 +5,7 @@
     reg price mpg rep78 headroom , coefl
 
 * Transpose and store the output
-    matrix results = `r(table)'
+    matrix results = r(table)
```
Comment (bbdaniels) [inline code/code.do]:
```suggestion
    matrix results = r(table)'
```

Comment (bbdaniels) [reply code/code.do]: @ankritisingh please update

**Comment by kbjarkefur** [comment]
Actually, lets close this branch. As after you change the base branch we have commits not related to this fix included in this pull request. 

@ankritisingh , we want to give you credit for spotting this, but we need to restart this fix. Can you please do exactly the same edit, but here (directly into the develop branch): https://github.com/worldbank/d4di/blob/develop/code/code.do

**Comment by kbjarkefur** [comment]
@ankritisingh, we are making a push on completing the next version of the book now. If you want to get credit for spotting this, then we need you to update this [here](https://github.com/worldbank/d4di/blob/develop/code/code.do) the way Ben is suggesting above. Let us know if you have questions. By next week we will do this ourselves.

**Pull request closed without merging**